LaTeX/Bibtex.sublime-syntax

```diff
@@ -78,12 +79,13 @@ contexts:
           captures:
             0: punctuation.section.entry.end.bibtex
           pop: true
-        - match: '([a-zA-Z]+)\s*(\=)'
+        - match: '([a-zA-Z]+)(\s*(\=)\s*)'
```

Comment:
- We don't need to escape `=` in regex.
- I checked [how JSON uses `meta.mapping.json`](https://github.com/sublimehq/Packages/blob/ff909a6922f1407dc10a1424a28af1cf5ed3f126/JSON/JSON.sublime-syntax#L87-L94), but it looks like quite different from this PR. JSON's `meta.mapping.json` applies to the whole object but this PR only applies to `\s*=\s*` which is quite weird.

Reply:
JSON differes due to the approach being used to apply `meta.mapping` vs. `meta.mapping.[key|value]`. JSON just avoids complex patterns with capture groups to not break highlighting by intermediate newlines.

It just looks like this PR is just to adjust scopes rather than reworking the whole syntax to apply other match strategies.

The key value separator `:` is scoped `meta.mapping.json`, which is the same as we see here.

I am fine with it here.

Reply:
Thanks for the explanation.